plugins/engines/ospray/OSPRayScene.h

```diff
@@ -50,32 +50,33 @@ class OSPRayScene : public Scene
     bool commitLights() final;
 
     void commitSimulationData();
-    void commitVolumeData();
+    bool commitVolumeData();
```

Comment:
this return value is a bit obscure; I would imagine that it indicates a possible failure, but after checking the code that's not the case... I think a separate getter for rebuildScene is preferable

Reply:
Doesn't make it better imo. That functions is private anyways, I'll do that first and we can discuss if you like.